content/developer/contributing/coding-standard.md

```diff
@@ -20,22 +20,22 @@ We adopted this standard for all the usual reasons: our team is growing, and we
 
 ### PHP Version
 
-All code in `master` must work under PHP 5.4.
+All code in `master` must work under PHP 5.6 but we strongly recommend using 7.0.
 
-All code in `release/2.2` must work under PHP 5.3.
+All code in `release/2.3` must work under PHP 5.4.
 
-We currently plan to move to PHP 5.5 after the fork for 2.3. These plans may change with little notice.
+For up to date information you can look at our [self-hosting-requirements](https://github.com/vanilla/vanilla/tree/master#self-hosting-requirements) section.
 
-### Check out CodeSniffer
+### Validating your PHP code with PHP_CodeSniffer
```

Comment:
Can this stay as "CodeSniffer"? It's "PHP_CodeSniffer" here, but "CodeSniffer" multiple times in the following lines.